app.js

```diff
@@ -0,0 +1,78 @@
+const options = ["rock", "paper", "scissors"];
+
+function getComputerChoice() {
+  const choice = options[Math.floor(Math.random() * options.length)];
```

Comment:
No need to assign it in a variable if you are returning it immediatly.

`return options[Math.floor(Math.random() * options.length)];`